packages/augur-core/source/contracts/trading/ZeroXTrade.sol

```diff
@@ -69,13 +72,15 @@ contract ZeroXTrade is Initializable, IZeroXTrade, IERC1155 {
     // solhint-disable-next-line var-name-mixedcase
     bytes32 public EIP712_DOMAIN_HASH;
 
+    IAugur augur;
```

Comment:
See comment below about deletion, but if you keep it this should have an explicit visibility modifier.